### ISSUE TITLE
USWDS - Usability Testing: Input mask findings

### ISSUE THREAD

**Issue by mahoneycm** [uswds/uswds#5525]
## Description

Input mask findings, recommendations, and next steps from the [Aug-Sep 2023 Usability Testing Findings Report 🔒](https://docs.google.com/document/d/1z1BRxYZq_m65IdsRKsMzDYSpYYHKsqYgelQe7w_qDUQ/edit?usp=sharing)

## Findings takeaways

When participants entered incorrect characters, they were not given proper feedback (4 users). They were not aware if the characters they were typing properly typed, or if they got deleted, and they were not made aware that they were unacceptable characters. One user said it was not apparent to her what characters were acceptable to begin with (U4).

[ Video clips](https://drive.google.com/file/d/13fcPNNPnmlT8y7GOf7NSzVMpe3ciCf5E/view?usp=drive_link):lock:
1. Not clear what characters are valid
1. No feedback after entering an invalid character (https://github.com/uswds/uswds/issues/5481)
1. Not clear how to correct after entering an invalid character (https://github.com/uswds/uswds/issues/5481)
1. Unclear status after entering an invalid character ("Did I actually enter the character?")

### 💡 COMPONENT original recommendations:

- The lack of feedback seems to be a known issue that we have an open PR for: [#5227](https://github.com/uswds/uswds/pull/5227).
- Provide audible feedback when incorrect format is typed.
- Continue work on [#5227](https://github.com/uswds/uswds/pull/5227)

### Finding 1: Feedback for incorrect characters

Previously reported in issue #5481 
Potentially resolved by: #5227 

Potentially related: [Notify users when maxlength has been reached #5482](https://github.com/uswds/uswds/issues/5482)

### Finding 2: Unaware of input criteria

Is there a way we can improve the AT user's understanding of the input pattern?

### Issues
- [ ] Add video clips to the issue
- [ ] #5227
